bigchaindb/assets.py

```diff
@@ -0,0 +1,78 @@
+import rethinkdb as r
+
+from bigchaindb.exceptions import AssetIdMismatch, TransactionDoesNotExist, AmountError
+
+
+def get_asset_id(txids, bigchain):
+    """Get the asset id from a list of transaction ids.
+
+    This is useful when we want to check if the multiple inputs of a transaction
+    are related to the same asset id.
+
+    Args:
+        txids (list): list of transaction ids.
```

Comment:
Yeah I don't know. This is just to make sure we don't just instantiate unnecessary bigchain instances.